map/src/context/AppContext.js

```diff
@@ -382,7 +410,10 @@ export const AppContextProvider = (props) => {
         weatherPoint, setWeatherPoint,
         routeTrackFile, setRouteTrackFile,
         searchCtx, setSearchCtx,
-        favorites, setFavorites
+        favorites, setFavorites,
+        planRoute, setPlanRoute,
```

Comment:
2 objects needed
currentlyEditTrack - null or object { }
localEditedTracks - [ ] - array of local edited objects with flag on / of

currentlyEditTrack.prototype.deletePointMap(ind) = function () {};
currentlyEditTrack.prototype.deletePoint(ind) = function... ( ... this.deletePointMap(ind); ... )

currentlyEditTrack.prototype.clearMap(ind) = function () {};
currentlyEditTrack.prototype.clear() = function... ( ... this.clearMap(); ... )